docs/TROUBLESHOOTING.md

```
@@ -0,0 +1,27 @@
+# 트러블슈팅
+
+---
+
+## HikariCP 커넥션 유효성 검증 실패
+
+**증상**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)